tools/toolchain/scripts/stage3/install_libxc.sh

```diff
@@ -29,8 +29,8 @@ case "$with_libxc" in
             if [ -f libxc-${libxc_ver}.tar.gz ] ; then
                 echo "libxc-${libxc_ver}.tar.gz is found"
             else
-                download_pkg ${DOWNLOADER_FLAGS} ${libxc_sha256} \
-                             https://www.cp2k.org/static/downloads/libxc-${libxc_ver}.tar.gz
+                download_pkg -o libxc-${libxc_ver}.tar.gz ${DOWNLOADER_FLAGS} ${libxc_sha256} \
```

Comment:
Please continue to download from our own server: https://www.cp2k.org/static/downloads/libxc-5.1.0.tar.gz
... because any external dependency can bring down our CI.

Reply:
Thank you for arranging it.